tests/unit/test_xnat_utils.py

```diff
@@ -61,3 +63,26 @@ def test_filter_scan_seqlist_discontinuity():
     expected_result = [x for x in data if int(x[0]) in seqlist]
 
     assert result == expected_result
+
+
+def test_unauthorized_user_exception_handling():
```

Comment:
This is great! I would also test that XNAT is reachable? Or wrong URL or project. 

This test won't pass on CI though... XNAT is behind VPN.

I am asking to enable internal runner for this org, but it may take a bit. So - if you want green tests, you may first check XNAT, and it reachable run the test, Else assert only something about the error when host is not reachable (404 HTTP)

Reply:
I've made the requested changes, creating Issue #81. However, testing for a wrong URL / project requires a valid username and password to be provided, or else 401 'Unauthorized' will be raised first—404 only being raised after those credentials have been verified. Do we have a generic username/password set that can be used solely for testing purposes? If so, I can add that test to integration, as well.

Reply:
Yes, we have a test user... let me see the credentials, since you are only running locally, you could just use yours in the `.env` file just like you do for the other integration tests